src/bandmap.c

```diff
@@ -447,20 +447,17 @@ void bandmap_age() {
  *
  * \return true if new multi
  */
-bool bm_ismulti(char *call, spot *data, int band) {
+bool bm_ismulti(spot *data, int band) {
```

Comment:
maybe even `band` is not needed if it's always called with `data->band`.

Reply:
Good idea.